src/Finder/LocatorInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Finder;
+
+/**
+ * @internal
+ */
+interface LocatorInterface
+{
+    /**
+     * Determine the realpath of the file by the file path.
+     *
+     * @param string $name
+     *
+     * @return string
+     */
+    public function locate(string $name): string;
+
+    /**
+     * Determine the realpath for the bunch of file path.
+     *
+     * @param array $fileNames
+     *
+     * @return string
+     */
+    public function locateAnyOf(array $fileNames): string;
```

Comment:
Perhaps `locateOneOf()` would be better? After all, the method returns a single path, not an arbitrary number.

Reply:
Maybe even `locateFirstOf()`?

Reply:
let stay with `oneOf`